backend/routes/event.js

```diff
@@ -85,9 +86,29 @@ router.delete("/:id", auth, (req, res, next) => {
     });
 });
 
-router.get('*', function(req, res){
-  res.status(500).json('bad request or 404 not found');
-});
+// router.get('*', function(req, res){
+//   res.status(500).json('bad request or 404 not found');
+// });
+
 // Should we allow updation to events, requires disussion
 
+router.put("/:id", auth, (req, res) => { 
+    const event = Event.findOne({ _id: req.params.id })
```

Comment:
use findOneAndUpdate instead
also, add a boolean to the schema named isUpdated and set it to true here (default false)

Reply:
Tested it with `findOneAndUpdate`, there was some retundency so, settled for this.